src/app/main/component/user/directives/notific-content-replace.directive.ts

```diff
@@ -45,12 +45,17 @@ export class NotificContentReplaceDirective implements OnChanges {
             userId: replacements.actionUserId[index]
           });
         }
+      } else if (contentKey === 'message' && replacements.notificationType) {
```

Comment:
The `replaceContent` function is currently too complex.
By focusing on task separation and using helper functions for different replacement logics, 
we have achieved a more elegant solution that is easier to extend in the future.